src/users/user.controller.ts

```diff
@@ -0,0 +1,55 @@
+import { Controller, Get, Param } from '@nestjs/common';
```

Comment:
seems those api routes wont be used for the moment. I think we should remove this

Reply:
Okay , maybe i'll put the create user in this route instead. It's more logic. But i'll remove the getters.